roles/installer/templates/deployments/task.yaml.j2

```
@@ -382,6 +387,8 @@ spec:
             - name: awx-devel
               mountPath: "/awx_devel"
 {% endif %}
+{% if redis_extra_volume_mounts -%}
+            {{ redis_extra_volume_mounts | indent(width=12, first=True) }}
 {% if termination_grace_period_seconds is defined %}
```

It looks like the redis_extra_env is missing here:

```suggestion
{% if redis_extra_env -%}
            {{ redis_extra_env | indent(width=12, first=True) }}
{% endif %}  
{% if termination_grace_period_seconds is defined %}
```

---

merge conflict needs to be resolved here

---

@rakesh561 This is still needed